tests/src/certs.c

```diff
@@ -25,6 +25,8 @@
 
 #include "mbedtls/legacy_or_psa.h"
 
+#include "pk_wrap.c"
```

Comment:
This should also be `mbedtls/pk.h`. (Note: I'm assuming `.c` was a typo!)

Reply:
sorry, this was an "impressive" (not in a good sense) series of errors all at once :/